agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -194,7 +198,13 @@ func TestStreamResources_Server_ActiveSecretValidation(t *testing.T) {
 		srv, store := newTestServer(t, nil)
 
 		// Write a seed peering.
-		require.NoError(t, store.PeeringWrite(1, tc.seed))
+		if tc.seed != nil {
+			require.NoError(t, store.PeeringWrite(1, &pbpeering.PeeringWriteRequest{Peering: tc.seed.peering}))
+
+			for _, s := range tc.seed.secrets {
```

Comment:
This is a slice because of there not being a method to insert a batch of peering secrets as-is outside of the restore function. Each step in the slice goes through secrets writes as they would in production: generate token, exchange secret, then promote pending.